sirepo/job_api.py

```diff
@@ -135,7 +141,7 @@ def api_runCancel(self):
         # Always true from the client's perspective
         return self.reply_json({"state": "canceled"})
 
-    @sirepo.api.Spec("require_user")
+    @sirepo.api.Spec("require_user", data="SimData all_input")
```

Comment:
RunMultiSpec (no need for all_input)

Reply:
9d450384bbf20294b6007b678c3dde0143022f40